packages/pigeon/lib/generator_tools.dart

```diff
@@ -497,3 +497,29 @@ Iterable<NamedType> getFieldsInSerializationOrder(Class klass) {
   // This returns the fields in the order they are declared in the pigeon file.
   return klass.fields;
 }
+
+/// Enum to specify which file will be generated for multi-file generators
+enum FileType {
+  /// header file.
+  header,
+
+  /// source file.
+  source,
+
+  /// file type is not applicable.
+  na,
+}
+
+/// Options for [Generator]'s that have multiple output file types.
```

Comment:
Nit: `[Generator]s` since it's a plural, not a possessive.

Reply:
oops